server/src/test/resources/csars/yaml/valid/lamp-input/my_app/create_myphpapp.sh

```diff
@@ -1,7 +1,4 @@
 #!/bin/bash
-# install php on a linux machine with php-mysql
```

Comment:
This is the installation of php not apache and it is mandatory because no where else is stated that php has to be installed on the websever

Reply:
Well i failed there (kind of). In order to simplify the transformation on kubernetes (the installation within the webapp create script causes issues) id suggest putting this in the Create Apache script.

Reply:
It actually was a duplicate. just look at the install apache script. 
![image](https://user-images.githubusercontent.com/2706976/34576717-22bd5a8e-f17f-11e7-9aea-6a2c8c433487.png)

However i dont like the idea that apache is only installed implicitly (if it even is)

Reply:
You are right that code is also in `install_apache.sh` but this file is never referenced in the template.yaml .
I actually like the idea that apache is installed implicitly. The WebServer.Apache node means that there should be a webserver running with apache installed.